docs/cli.rst

```diff
@@ -1,11 +1,248 @@
+
+.. code-block:: console
+
+    $ cci task
+    Usage: cci task [OPTIONS] COMMAND [ARGS]...
+
+    Commands for finding and running tasks for a project
+
+    Options:
+    --help  Show this message and exit.
+
+    Commands:
+    doc   Exports RST format documentation for all tasks
+    info  Displays information for a task
+    list  List available tasks for the current context
+    run   Runs a task
+
+We see that there are even more subcommands available under ``cci task``.
+
+Getting ``--help``
+------------------
+If you aren't sure what a specific command does you can always utilize the ``--help`` option.
+If we want help with running a task we could type ``cci task run --help``:
+
+.. code-block:: console
+
+    $ cci task run --help
+    Usage: cci task run [OPTIONS] TASK_NAME
+
+    Runs a task
+
+    Options:
+    --org TEXT      Specify the target org.  By default, runs against the
+                    current default org
+
+    -o TEXT...      Pass task specific options for the task as '-o option
+                    value'.  You can specify more than one option by using -o
+                    more than once.
+
+    --debug         Drops into pdb, the Python debugger, on an exception
+    --debug-before  Drops into the Python debugger right before task start.
+    --debug-after   Drops into the Python debugger at task completion.
+    --no-prompt     Disables all prompts.  Set for non-interactive mode use such
+                    as calling from scripts or CI systems
+
+    --help          Show this message and exit.    
+
+This gives us a clear usage statement, description, and shows all options available for use with the command.
+
+
+Working with Tasks and Flows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Listing Tasks and Flows
+****************************
+``cci`` ships with many standard tasks and flows.
+``cci`` has two commands for listing available tasks and flows:
+
+.. code-block:: console
+
+    $ cci task list
+    $ cci flow list
+
+The tasks and flows listed are *specific to the project* you're running the command int.
+If you have a custom flow defined in your ``cumulusci.yml`` file for ProjectA, it will only show if you run ``cci flow list`` in ProjectA's repository directory.
+Tasks and Flows are listed grouped by their ``group`` attribute as specified in the ``cumulusci.yml`` file.
+This means it's easy to edit these groups as you see fit!
+Any changes made will be reflected in these commands.
+
+Running Tasks and Flows
+*******************************
+Once you know the specifc task or flow you want to run, they can be executed with the ``run`` command:
+
+.. code-block:: console
+
+    $ cci task run task_name
+    $ cci flow run flow_name
+
+Where ``task_name`` and ``flow_name`` are the actual name of the task or flow that you would like to run.
+Tasks usually require additional options to be passed when using the ``cci task run`` command.
+See the next section for how to view task specific option information. 
```

Comment:
```suggestion
See the next section for how to view task-specific option information. 
```

Reply:
It's also usually necessary to specify an org when running a task or flow. We should probably mention this in some way although we need to figure out the proper sequencing between this and the sections on orgs.